File: data/museum/oseg_generated/typescript-node/GetTicketCode_default_example.ts
import * as fs from 'fs';
import * as openapimuseum_client from "openapimuseum_client";

const apiCaller = new openapimuseum_client.TicketsApi();

const ticketId = "a54a57ca-36f8-421b-a6b4-2e8f26858a4c";
apiCaller.getTicketCode(
    ticketId,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling Tickets#getTicketCode:");
  console.log(error.body);
});

File: data/museum/oseg_generated/ruby/GetTicketCode_default_example.rb
require "openapimuseum_client"

OpenapiMuseumClient.configure do |config|
end

begin
    api_caller = OpenapiMuseumClient::TicketsApi.new

    response = api_caller.get_ticket_code(
        "a54a57ca-36f8-421b-a6b4-2e8f26858a4c",
    )

    FileUtils.cp(response.path, "path/to/file.zip")
rescue OpenapiMuseumClient::ApiError => e
    puts "Exception when calling Tickets#get_ticket_code: #{e}"
end
